proto/brpb.proto

```diff
@@ -236,6 +237,27 @@ message GCS {
     string credentials_blob = 6;
 }
 
+// Azure storage backend saves files into azure blob storage.
+message Azure {
+    string endpoint = 1;
+    // Alias: container
+    string bucket = 2;
+    // Notice: prefix starts without `/`, otherwise the first directory's name is empty.
+    string prefix = 3;
+    // Alias: access_tier. If empty, set `Hot` as default
+    // See https://docs.microsoft.com/en-us/azure/storage/blobs/access-tiers-overview
+    string storage_class = 4;
+
+    string account_name = 5;
+    // Use shared key to access the azure blob
+    // If empty and use_bearer_token is false, read shared key in the environment.
+    string shared_key = 6;
+    // Prefer to use token to access the azure blob
+    // Need variables in environment: $AZURE_CLIENT_ID, $AZURE_TENANT_ID, $AZURE_CLIENT_SECRET
+    // See https://docs.microsoft.com/en-us/azure/storage/common/identity-library-acquire-token?toc=/azure/storage/blobs/toc.json
+    bool use_bearer_token = 7;
```

Comment:
🤔 how do we define this flag? i think it's better for TiKV to implicitly determine from their nodes' environment.

Reply:
Done. Please take another look.